app/components/provider_interface/application_timeline_component.rb

```diff
@@ -53,7 +53,7 @@ def note_events
         application_choice.notes.order('created_at').map do |note|
           Event.new(
             'Note added',
-            provider_name(note.provider_user),
+            note.provider_user.full_name,
```

Comment:
should we switch this up to `user`?

Reply:
@cwrw updated https://github.com/DFE-Digital/apply-for-teacher-training/pull/6360/commits/4de3f9b228b6821f7c4515ed4b0b0a4c203933a6